### DISCUSSION TITLE
Are blank values in tokens meant to pass ValidateOptions?

### DISCUSSION THREAD

**Discussion by carwyn** [lestrrat-go/jwx#353] [Q&A] [answered]
In `jwt/validate.go` there are a series of checks like this:

```
	if len(issuer) > 0 {
		if v := t.Issuer(); v != "" && v != issuer {
			return errors.New(`iss not satisfied`)
		}
	}
````

Doesn't this mean that an empty string in the token `t.Issuer()` (aka `v`) satisfies in all cases even if `issuer` is set due to the `v != ""`?

**Comment by lestrrat** [comment, marked as answer]
Huh, yeah that looks weird. I guess I wanted to check something about the value being not empty, but... yeah.

**Reply by lestrrat** [reply]
I jotted a fix, but I need to jet and take care of things, so please wait for a release. https://github.com/lestrrat-go/jwx/pull/354

**Reply by lestrrat** [reply]
done, released v1.1.5
